internal/ingress/controller/nginx.go

```diff
@@ -87,9 +88,10 @@ func NewNGINXController(config *Configuration, mc metric.Collector) *NGINXContro
 	n := &NGINXController{
 		isIPV6Enabled: ing_net.IsIPv6Enabled(),
 
-		resolver:        h,
-		cfg:             config,
-		syncRateLimiter: flowcontrol.NewTokenBucketRateLimiter(config.SyncRateLimit, 1),
+		resolver:         h,
+		cfg:              config,
+		syncRateLimiter:  flowcontrol.NewTokenBucketRateLimiter(config.SyncRateLimit, 1),
+		workersReloading: false,
```

Comment:
Shouldn't the default be set to config.WorkerSerialReloads

Reply:
No. This is a state flag, not a config flag

Reply:
The workersReloading flag controls the state of whether there is a reload going on, or not, and then if WorkerSerialReloads is enabed, and workersReloading is true, it will not start a new reload, but rather re-queue it. From the start, there is no reload happening so this is set to false.